tobac/tracking.py

```diff
@@ -239,21 +240,36 @@ def fill_gaps(
 
 def add_cell_time(t):
     """add cell time as time since the initiation of each cell
-    Input:
+
+    Parameters
+    ----------
     t:             pandas  DataFrame
                    trajectories with added coordinates
-    Output:
+
+    Returns
+    -------
     t:             pandas dataframe
                    trajectories with added cell time
     """
 
-    logging.debug("start adding time relative to cell initiation")
+    # logging.debug('start adding time relative to cell initiation')
     t_grouped = t.groupby("cell")
-    t["time_cell"] = np.nan
-    for cell, track in t_grouped:
-        track_0 = track.head(n=1)
-        for i, row in track.iterrows():
-            t.loc[i, "time_cell"] = row["time"] - track_0.loc[track_0.index[0], "time"]
-    # turn series into pandas timedelta DataSeries
+
+    t["time_cell"] = t["time"] - t.groupby("cell")["time"].transform("min")
     t["time_cell"] = pd.to_timedelta(t["time_cell"])
     return t
+
+
+def remap_particle_to_cell_nv(particle_cell_map, input_particle):
```

Comment:
Would it be better to make this as a lambda function in or just before line 124 for clarity?

Reply:
Good point. I've switched this to a lambda function now.